requirements.txt

```
@@ -18,3 +18,5 @@ tldextract
 python-dateutil
 markupsafe>=2.1.1
 ua_parser
+Py3AMF; python_version >= '3.0'
+pyAMF; python_version < '3.0'
```

```suggestion
py3AMF
```

pywb no longer runs on Python 2, so no need for these conditionals anymore